basics.js

```diff
@@ -3,33 +3,42 @@
 // prefixed with the text "I have a " (so "I have a cat", "I have a dog", etc.)
 
 var pets = ['cat', 'dog', 'wombat'];
+function animalArray(){
+  for (var i=0; i < pets.length; i++){
+    console.log("I have a " + pets[i]);
+  }
+};
+animalArray();
 
 ///// Question 2
 // Change my friend's hair color to "green", without modifying the line below.
 
 var friend = { name: 'Chris', age: 27, hairColor: 'blue' };
+friend.haircolor = null;
```

Comment:
Two things:

1) We probably want to set it to 'green' instead of `null`. 
2) Make sure to keep an eye out for camelCase, as its hairColor instead of haircolor.